.github/README.md

```diff
@@ -84,13 +84,13 @@ Now compile exec.c:
 kcc exec.c
 ```
 
-Go to src/user and generate `exec.skel.h` with the following command:
+Now, still from src/kern,  generate `exec.skel.h` with the following command:
```

Comment:
A best-practice with git (or any source control) is to keep changes "atomic" (minimal). Ideally, each commit does one thing and only one. There are good reasons for this. One reason is that I might be looking at the project git log in search of a commit that could have introduced this change, and if I see "hawk: Add counter feature using PERCPU_ARRAY map" I will instantly assume that it did not change that line because that seems unrelated. Another reason is that someday we might discover that this PERCPU_ARRAY caused a massive bug and we want to revert the code asap. If each commit is atomic, we can easily use `git revert commitHash` and drop only the feature that caused the bug. But in that case, if we were to revert the PERCPU_ARRAY we would also loose the README refactoring.

One last tip to make this easier. You can use "git add -p README.md" to be prompted with an interactive tool that lets you pick what changes in the file go into the next commit. For example, if you have already a README.md with changes all around, you can use `git add -p` to select the refactoring once, then git commit and then later create a new commit with what remains.